extensionsGallery-insider.json

```diff
@@ -75,14 +75,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.1.3",
-							"lastUpdated": "11/22/2019",
+							"version": "0.3.0",
```

Comment:
mmm, I'd rather not we do this to avoid confusion. It's minor but it's only a couple minutes of work to change the version back.

I'd also rather see us just stick to a minor version bump. There's no additional features or major changes being made here (just some shifting around of what tabs stuff lives in) so I think we should follow typical semantic versioning guidelines and just have this be a minor version bump.

Reply:
done, please take a look at this one: https://github.com/microsoft/azuredatastudio/pull/10176
i have also uploaded the new packages.